pkg/cmd/issue/create/create.go

```diff
@@ -157,22 +157,20 @@ func createRun(opts *CreateOptions) (err error) {
 
 	tpl := shared.NewTemplateManager(httpClient, baseRepo, opts.RootDirOverride, !opts.HasRepoOverride, false)
 
+	var openURL string
+
 	if opts.WebMode {
-		openURL := ghrepo.GenerateRepoURL(baseRepo, "issues/new")
 		if opts.Title != "" || opts.Body != "" || tb.HasMetadata() {
-			if len(opts.Projects) > 0 {
-				var err error
-				tb.Projects, err = api.ProjectNamesToPaths(apiClient, baseRepo, tb.Projects)
-				if err != nil {
-					return fmt.Errorf("could not add to project: %w", err)
-				}
-			}
-			openURL, err = prShared.WithPrAndIssueQueryParams(openURL, tb)
+			openURL, err = generatePreviewURL(apiClient, baseRepo, tb)
 			if err != nil {
 				return
 			}
+			if !utils.ValidURL(openURL) {
+				err = fmt.Errorf("cannot open in browser: maximum URL length exceeded")
+				return
+			}
 		} else if ok, _ := tpl.HasTemplates(); ok {
-			openURL += "/choose"
+			openURL = ghrepo.GenerateRepoURL(baseRepo, "issues/new/choose")
```

Comment:
I have been traveling for work and have not received a response from you yet because we usually need to get them in a new way for the updates.